src/main/java/org/gridsuite/voltageinit/server/dto/VoltageInitStatus.java

```diff
@@ -12,5 +12,15 @@
 public enum VoltageInitStatus {
     NOT_DONE,
     RUNNING,
-    COMPLETED
+    COMPLETED,
+
+    /**
+     * {@link com.powsybl.openreac.parameters.output.OpenReacStatus#OK}
+     */
+    OK,
```

Comment:
`COMPLETED` and `OK` look very similar. It is a bit confusing to have both. Maybe add a little comment for clarity to explain why thay are needed and when they are used ?